### PR TITLE
[1LP][NOTEST][RFR] This PR will add wait between validate and save operation.

### PR DESCRIPTION
**Pull request by sbulage**

### CODE REVIEW

File: cfme/tests/cloud_infra_common/test_vm_instance_analysis.py [hunk 1]
```diff
@@ -167,7 +168,9 @@ def set_agent_creds(appliance, request, provider):
     ssa_agent_view = view.browser.create_view(EC2EndpointForm)
     ssa_agent_view.smart_state_docker.fill({'username': username, 'password': password})
     ssa_agent_view.default.validate.click()
+    time.sleep(35)
```
Comment (sbulage) [inline cfme/tests/cloud_infra_common/test_vm_instance_analysis.py]: After validation, we have to wait for approx.30-35 sec.  for credentials to be validated from docker.io. Tried all possible conditions which can be use in `wait_for` but not showing required output hence `time.sleep` is added.

Comment (rlbabyuk) [reply cfme/tests/cloud_infra_common/test_vm_instance_analysis.py]: @sbulage please add bz, or explanation that we discussed

Comment (sbulage) [reply cfme/tests/cloud_infra_common/test_vm_instance_analysis.py]: @rlbabyuk yeah sure, I will add it after discussion with devel.

File: cfme/tests/cloud_infra_common/test_vm_instance_analysis.py [hunk 2]
```diff
@@ -167,7 +168,10 @@ def set_agent_creds(appliance, request, provider):
     ssa_agent_view = view.browser.create_view(EC2EndpointForm)
     ssa_agent_view.smart_state_docker.fill({'username': username, 'password': password})
     ssa_agent_view.default.validate.click()
+    if BZ(1608023, forced_streams=['5.9']).blocks:
```
Comment (rlbabyuk) [inline cfme/tests/cloud_infra_common/test_vm_instance_analysis.py]: Question: How about 5.10? is it working ok

File: cfme/tests/cloud_infra_common/test_vm_instance_analysis.py [hunk 3]
```diff
@@ -167,7 +168,10 @@ def set_agent_creds(appliance, request, provider):
...
+        time.sleep(35)
     view.save.click()
+    view.browser.refresh()
```
Comment (rlbabyuk) [inline cfme/tests/cloud_infra_common/test_vm_instance_analysis.py]: Question: why we need this view.browser.refresh() here

Comment (sbulage) [reply cfme/tests/cloud_infra_common/test_vm_instance_analysis.py]: @rlbabyuk After saving creds, it won't reflect immediately and navigate to other page.

**Comment by psav** [comment]
We REALLY need a comment in here stating how bad this is :) As well as the BZ :)

File: cfme/tests/cloud_infra_common/test_vm_instance_analysis.py [hunk 4]
```diff
@@ -171,7 +172,11 @@ def set_agent_creds(appliance, request, provider):
     ssa_agent_view = view.browser.create_view(EC2EndpointForm)
     ssa_agent_view.smart_state_docker.fill({'username': username, 'password': password})
     ssa_agent_view.default.validate.click()
+    # TODO: Remove time.sleep asap BZ fix.
+    if BZ(1608023, forced_streams=['5.9', '5.10']).blocks:
+        time.sleep(35)
```
Comment (mshriver) [inline cfme/tests/cloud_infra_common/test_vm_instance_analysis.py]: gross.

Comment (sbulage) [reply cfme/tests/cloud_infra_common/test_vm_instance_analysis.py]: :(

Comment (mshriver) [reply cfme/tests/cloud_infra_common/test_vm_instance_analysis.py]: I've had to do it in the auth tests too, at least its wrapped in a BZ.

**Pull request merged**